site/public/js/gradeable-notebook.js

```diff
@@ -79,13 +79,49 @@ function notebookAutosaveKey() {
  * Saves the current state of the notebook gradeable to localstorage.
  */
 function saveNotebookToLocal() {
-    if (typeof autosaveEnabled !== 'undefined' && autosaveEnabled) {
-        localStorage.setItem(notebookAutosaveKey(), JSON.stringify({
-            timestamp: Date.now(),
-            multiple_choice: gatherInputAnswersByType('multiple_choice'),
-            codebox: gatherInputAnswersByType('codebox'),
-        }));
-    }
+    const mc_inputs=[];
+    const short_answer_inputs = [];
+
+    //loop through multiple choice questions and save answers
+    $('.multiple_choice').each(function(){
+        let file_name='';
+        $(this).children('fieldset').each(function(){
+            file_name = $(this).data('filename');
+            if (file_name) {
+                const answers = [];
+                $(this).children('label').each(function(){
+                    //grab selected answers
+                    $(this).children('input').each(function(){
+                        if ($(this)[0].checked) {
+                            answers.push($(this)[0].defaultValue);
+                        }
+                    });
+                });
+                mc_inputs.push([file_name, answers]);
+            }
+        });
+    });
+
+    //save short answers
+    $('.short_answer').each(function(){
+        $(this).children('div').each(function(){
+            let file_name = $(this).data('filename');
+            if (file_name) {
+                let value = '';
+                file_name = file_name.trim();
+                //grab input
+                const editor = ($(this)[0]).querySelector('.CodeMirror').CodeMirror;
+                value = editor.getValue();
```

Comment:
```suggestion
            const file_name = $(this).data('filename').trim();
            if (file_name) {
                //grab input
                const editor = $(this).find('.CodeMirror').CodeMirror;
                const value = editor.getValue();
```
Just a few changes to use const where possible. 
Also, you can use `find` as a JQuery alternative to `querySelector` which removes the need to convert to an HTMLElement first.